bugbug/repository.py

```diff
@@ -20,6 +22,14 @@
 BUG_PATTERN = re.compile('[\t ]*[Bb][Uu][Gg][\t ]*([0-9]+)')
 
 
+def hg_log(repo_dir, template=templates.changeset):
+    hg = hglib.open(repo_dir)
+    args = cmdbuilder(b'log', template=template)
+    x = hg._parserevs(hg.rawcommand(args))
```

Comment:
Implement your own parsing here, as our custom template doesn't have the same values as the default template.

Reply:
> Implement your own parsing here, as our custom template doesn't have the same values as the default template.

Ah, sorry, I forgot about this.